quantum-adapters/kiwoom-adapter/src/kiwoom_api/functions/auth.py

```
@@ -0,0 +1,165 @@
+            from ..models.auth import CachedTokenInfo
+        except ImportError:
+            from kiwoom_api.models.auth import CachedTokenInfo
+
+        # 만료시간 파싱 (키움 API 형식: YYYYMMDDHHMMSS)
+        expires_dt = body.get('expires_dt', '')
+        try:
+            if expires_dt and len(expires_dt) == 14:
+                expires_at = datetime.strptime(expires_dt, "%Y%m%d%H%M%S")
+            else:
+                expires_at = datetime.now() + timedelta(hours=24)  # 기본값
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)